### ISSUE TITLE
Q: Input type File ( for uploading) 

### ISSUE THREAD

**Issue by lstraka-telekom** [telekom/scale#697]
Hi, 

Do you have in your backlog also included new input type for files ? single/multi files etc ?

**Comment by acstll** [comment]
Yes, indeed. A files upload components is in the backlog. We even worked on it but had to prioritise other work.

**Comment by tshimber** [comment]
@amir-ba i guess we should keep only feature tag and leave it open. maybe someday?..

**Comment by amir-ba** [comment]
> @amir-ba i guess we should keep only feature tag and leave it open. maybe someday?..

We will keep feature request open, but they will not be pioratized now.
